MapboxNavigation/NavigationMapView.swift

```diff
@@ -46,6 +46,14 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
      */
     public var tapGestureDistanceThreshold: CGFloat = 50
     
+    /**
+     A collection of road classes for which a congestion level substitution should occur.
+     
+     For any road class included in the `trafficOverrideRoadClasses`, all route segments with an `.unknown` traffic congestion level and a matching `MapboxDirections.RoadClass`
+     will be replaced with the `.low` congestion level.
+     */
+    public var trafficOverrideRoadClasses: [RoadClasses] = []
```

Comment:
RoadClasses is an option set, which means you can already specify multiple road classes at a time. If there’s no need to distinguish between, say, tolled motorways and other toll roads, then `RoadClasses` is a sufficient type.

Is there any need to explicitly state the substitution that should occur by making this property a dictionary of RoadClasses to RoadClasses?

Reply:
Or is the intention to preserve the order of the intersections in a parallel data structure?

Reply:
This is actually a possible subject of a change. Since Android implementation is currently using road classes from [mapbox-streets-v8](https://docs.mapbox.com/vector-tiles/reference/mapbox-streets-v8/) and iOS is does not, there is a slight difference between road classes in certain road segments. 

After `Road Classification` implementation lands in Mapbox Directions we should revise this public property: e.g. in case if we decide to extend `RoadClasses` with classes from `mapbox-streets-v8` current property can look like you mention:
```suggestion
    public var trafficOverrideRoadClasses: RoadClasses
```

If we plan to add new entity for `mapbox-streets-v8` road classes we'll have to replace current property with it (either it's bitmask, dictionary or array will depend on implementation).

Reply:
For now I've replaced `[RoadClasses]` array to be RoadClasses option set. Will revise once again when Road Classification feature is implemented.